boucle.py

```diff
@@ -4,18 +4,19 @@
 
 import jack
 import sys
-from clip import Clip, Song
+from clip import Clip, Song, getDefaultOutputNames
 from gui import Gui
 from PyQt5.QtWidgets import QApplication
 from queue import Empty
 
-song = Song(8, 8)
+song = Song(8, 8, getDefaultOutputNames(8))
 
 client = jack.Client("Super Boucle")
 midi_in = client.midi_inports.register("input")
 midi_out = client.midi_outports.register("output")
-outL = client.outports.register("output_L")
-outR = client.outports.register("output_R")
+outL = client.outports.register("MAIN_L")
+outR = client.outports.register("MAIN_R")
+
 inL = client.inports.register("input_L")
 inR = client.inports.register("input_R")
 
```

Comment:
@IARI @Vampouille Yes I won't use the Main port in my setup. Each instrument (mute group) will go out as a port an will be mixed elsewere (ardour, carla...).
For my point of user/musician/soundengineer view it is important to keep the ports static because I really need to keep all connections (after superboucle) during the whole concert. Maybe I might not route clips to one port in on song, but in the other straight away, so I need to keep the whole mixing setup static and not change anything to keep it stable (especially prevent autoconnection faults by the routing managers afterward...).